package.json

```diff
@@ -24,6 +24,7 @@
     "react-native-orientation": "^1.17.0",
     "react-native-router-flux": "^3.35.0",
     "react-native-simple-store": "^1.1.0",
+    "react-native-svg": "~4.3.3",
```

Comment:
**Note**: this initially caused some dependency issues when clearing node_modules and installing again (fixed with later commit). We should add a checklist for PRs to ensure things work on each branch after a new `npm install`.